lib/solidus_segment/serializers/line_item_serializer.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SolidusSegment
+  module Serializers
+    class LineItemSerializer
```

Comment:
This enhances the VariantSerialize, what do you think to inherit from it and add/override just the needed values?